database/postgres/postgres.go

```diff
@@ -121,10 +123,19 @@ func (p *Postgres) Open(url string) (database.Driver, error) {
 	}
 
 	migrationsTable := purl.Query().Get("x-migrations-table")
+	statementTimeoutString := purl.Query().Get("x-statement-timeout")
+	statementTimeout := 0
+	if statementTimeoutString != "" {
+		statementTimeout, err = strconv.Atoi(statementTimeoutString)
+		if err != nil {
+			return nil, err
```

Comment:
@dhui wondering if we should throw a custom error here instead of the Atoi one, thoughts?

Reply:
I think we'd lose information w/ a custom error, so it'd be cleaner to wrap the error using `multierror`.

Reply:
> Lemme know if you wanna wrap the `Atoi()` error using `multierror` and I'll hold off on merging until then.

not urgent. please go ahead and merge this when you can.